Chaincase.UI/wwwroot/style/global.css

```diff
@@ -1,3 +1,57 @@
+﻿/** Global **/
+* {
+    box-sizing: border-box;
```

Comment:
this is a pretty fat opinion. Why? Just preference?

Reply:
It's just useful. Especially when using things like `min-height` or `max-width`